components/appointments/id/index.tsx

```diff
@@ -112,160 +163,328 @@ export default function Appointment({ aid, session }: AppointmentProps) {
+                    value={format(
+                      new Date(appointment.date),
+                      'MMM d, yyyy - h:mm a'
+                    )}
+                    classNames={{ icon: 'text-yellow-500 bg-yellow-50' }}
                   />
-                </div>
-                <div className="">
                   <CellRenderer
-                    label="Name"
-                    icon="cbi:abc"
-                    value={appointment.doctor?.name}
-                    classNames={{ icon: 'text-pink-500 bg-pink-50' }}
-                    className="text-lg text-black"
+                    label="Appointment Type"
+                    icon="solar:stethoscope-bold"
+                    value={appointment.type}
+                    classNames={{
+                      icon: 'text-primary bg-primary-50',
+                    }}
+                  />
+                  <CellRenderer
+                    label="created by"
+                    icon="solar:people-nearby-bold"
+                    value={appointment.createdBy || 'System'}
+                    classNames={{
+                      icon: 'text-purple-500 bg-purple-50',
+                    }}
                   />
+                </div>
+                <div>
                   <CellRenderer
-                    label="Email"
-                    icon="material-symbols:mail"
-                    value={appointment.doctor?.email}
+                    label="Mode"
+                    icon="solar:hospital-bold"
+                    value={appointment.additionalInfo.type}
                     classNames={{ icon: 'text-blue-500 bg-blue-50' }}
-                    className="text-lg text-black"
                   />
                   <CellRenderer
-                    label="Phone Number"
-                    icon="ic:baseline-phone"
-                    value={appointment.doctor?.phone}
-                    classNames={{ icon: 'text-green-500 bg-green-50' }}
-                    className="text-lg text-black"
+                    label="Created At"
+                    icon="solar:clock-circle-linear"
+                    value={
+                      appointment.createdAt
+                        ? formatDate(new Date(appointment.createdAt), 'PPP')
+                        : 'N/A'
+                    }
+                    classNames={{ icon: 'text-pink-500 bg-pink-50' }}
                   />
                   <CellRenderer
-                    label="Sitting"
-                    icon="fluent:status-20-filled"
-                    value={appointment.doctor?.sitting}
-                    classNames={{ icon: 'text-purple-500 bg-purple-50' }}
-                    className="text-lg text-black"
+                    label="Updated At"
+                    icon="solar:sort-by-time-broken"
+                    value={
+                      appointment.updatedAt
+                        ? formatDate(new Date(appointment.updatedAt), 'PPP')
```

Comment:
same here use `format` instead of `formatDate`
```suggestion
                        ? format(new Date(appointment.updatedAt), 'PPP')
```